examples/todos/src/components/Todo.js

```diff
@@ -1,7 +1,14 @@
+// @flow
 import React from 'react'
-import PropTypes from 'prop-types'
 
-const Todo = ({ onClick, completed, text, createdTime }) => (
+type props = {
```

Comment:
Maybe we could change the name to `TodoType` or `PropsType`. More meaning than this one, right? :D